lib/inspec/resources/postgres.rb

```diff
@@ -4,6 +4,8 @@ module Inspec::Resources
   class Postgres < Inspec.resource(1)
     name "postgres"
     supports platform: "unix"
+    supports platform: "windows"
```

Comment:
I would assume `osx` is covered under `unix`?

Reply:
Yes